database/clickhouse/clickhouse_test.go

```diff
@@ -0,0 +1,184 @@
+package clickhouse
```

Comment:
Can this be `clickhouse_test`?

Reply:
this test included in the package clickhouse, not clichouse_test. Other databases tests included in databases packages too

Reply:
I didn't see any usage of unexported identifiers, but could have missed some. If we don't need to use unexported identifiers in tests, let's use a different package name for the tests and import the `clickhouse` package.

> Other databases tests included in databases packages too

Some db tests test unexported methods. We should really split our tests into internal and external tests...

Reply:
I understood. Thanks, fixed